.changeset/short-singers-serve.md

```diff
@@ -12,4 +12,4 @@ After Dec 14th, the fields will be removed from types and classes of the Backsta
 the first release after that, they will not be present in released packages either.
 
 If your catalog-info.yaml files still contain this field after the deletion, they will still be
-valid and your ingestion will not break, but they won't be visible in the types for consuming code, and the expected relations will not be generated based on them either.```
+valid and your ingestion will not break, but they won't be visible in the types for consuming code, and the expected relations will not be generated based on them either.
```

Comment:
Hm not sure you can edit others' changesets :)

Reply:
It's mine, but if you think it's an issue that it's included in this PR then I create a separate one

Reply:
I'm thinking it may mix up the PR linkage or so.

Reply:
Move this to #3468